Libplanet.Tests/Net/SwarmTest.cs

```diff
@@ -1712,28 +1710,37 @@ public async Task Preload()
                 minerSwarm.FindNextHashesChunkSize = 2;
                 await receiverSwarm.PreloadAsync(TimeSpan.FromSeconds(15), progress);
 
-                Assert.Equal(minerChain.AsEnumerable(), receiverChain.AsEnumerable());
+                Assert.True(ChainsEqual(minerChain, receiverChain));
 
-                PreloadState[] expectedStates = minerChain.Select((b, i) =>
+                var expectedStates = new List<PreloadState>();
+
+                for (var i = 0; i < minerChain.Count; i++)
                 {
-                    return new BlockDownloadState
+                    var b = minerChain[i];
+                    var state = new BlockDownloadState
                     {
                         ReceivedBlockHash = b.Hash,
                         TotalBlockCount = 10,
                         ReceivedBlockCount = i + 1,
                     };
-                }).ToArray();
-                (expectedStates[10] as BlockDownloadState).TotalBlockCount = 11;
+                    expectedStates.Add(state);
+                }
 
-                expectedStates = expectedStates.Concat(minerChain.Select(
-                    (b, i) => new ActionExecutionState()
+                ((BlockDownloadState)expectedStates[10]).TotalBlockCount = 11;
+
+                for (var i = 0; i < minerChain.Count; i++)
+                {
+                    var b = minerChain[i];
+                    var state = new ActionExecutionState
                     {
                         ExecutedBlockHash = b.Hash,
                         TotalBlockCount = 11,
                         ExecutedBlockCount = i + 1,
-                    })).ToArray();
+                    };
+                    expectedStates.Add(state);
+                }
 
-                Assert.True(expectedStates.ToImmutableHashSet()
+                Assert.True(expectedStates.ToArray().ToImmutableHashSet()
```

Comment:
`List<T>.ToImmutableHashSet()` seems enough to make `ImmutableHashSet<T>`, is there intention?

Reply:
Nope, fixed.